src/backend/distributed/planner/relation_restriction_equivalence.c

```diff
@@ -2050,22 +2042,26 @@ JoinRestrictionListExistsInContext(JoinRestriction *joinRestrictionInput,
 		 * If we're dealing with different queries, we shouldn't treat their
 		 * restrictions as being the same.
 		 */
-		if (joinRestriction->plannerInfo != joinRestrictionInput->plannerInfo)
+		if (joinRestrictionInContext->plannerInfo != joinRestrictionInTest->plannerInfo)
 		{
 			continue;
 		}
 
 		/*
-		 * We check whether the restrictions in joinRestriction is a super set
-		 * of the restrictions in joinRestrictionInput in the sense that all the
+		 * We check whether the restrictions in joinRestrictionInContext is a super
```

Comment:
As the function name is  `LeftListIsSubset `, should we  use `subset` in the comment  instead of `superset`?